app/backend/src/couchers/descriptor_pool.py

```diff
@@ -0,0 +1,21 @@
+import functools
+from pathlib import Path
+
+from google.protobuf import descriptor_pb2, descriptor_pool
+
+
+@functools.cache
+def get_descriptor_pool():
+    """
+    Generates a protocol buffer object descriptor pool which allows looking up info about our proto API, such as options
+    for each servicer, method, or message.
+    """
+    # this needs to be imported so the annotations are available in the generated pool...
+    from proto import annotations_pb2
```

Comment:
Side-effects on import really sucks, and autoflake wants to remove this unused import. Good with a comment.